proposals/victoriametrics.adoc

```diff
@@ -0,0 +1,140 @@
+== VitoriaMetrics project proposal
+
+*Name of project*
+
+VictoriaMetrics
+
+
+*Project description*
+
+VitoriaMetrics is open-source fast cost effective time series database with good vertical and horizontal scalability.
+It integrates well with Prometheus+Grafana ecosystem as long-term remote storage for Prometheus.
+It provides Prometheus-compatible querying API and it supports PromQL.
+
+_History_:
+
+- Current release: 1.19.2
+- May, 2019: single-node and cluster versions were open-sourced. https://blog.usejournal.com/open-sourcing-victoriametrics-f31e34485c2b
+- January, 2019: the first production use
+- December, 2018: data storage format has been stabilized
+- October, 2018: public announcement of SaaS version. https://medium.com/faun/victoriametrics-creating-the-best-remote-storage-for-prometheus-5d92d66787ac
+- January, 2018: initial commit
+- October, 2017: architecture design and prototype
+
+More info is available by the following links:
+
+- https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md
+- https://github.com/VictoriaMetrics/VictoriaMetrics/wiki/Articles
+
+
+*Statement on alignment with CNCF charter mission*
+
+VictoriaMetrics is aligned well with CNCF because of:
+
+- It is easy to configure and operate as long-term remote storage for Prometheus Operator. It may collect Prometheus data from multiple Kubernetes clusters.
```

Comment:
Prometheus Operator is a tool to run Prometheus on Kubernetes. Do you mean “Prometheus server” or ”Prometheus server running on Kubernetes”?

Reply:
Clarified the meaning of this sentence.